### ISSUE TITLE
Process `<template strip-whitespace>` ahead of time

### ISSUE THREAD

**Issue by azakus** [Polymer/vulcanize#396]
It should be possible, and a nice optimization, to process the `strip-whitespace` directive for `<template>` elements ahead of time.

**Comment by justinfagnani** [comment]
What do you mean ahead of time? Ahead of what time? Vulcanize?

**Comment by azakus** [comment]
Yes, vulcanize could find templates with `strip-whitespace` and remove whitespace nodes while inlining HTML imports.
